frontend/src/apis/infomed.tsx

```diff
@@ -133,10 +133,23 @@ export const fetchHospitalInventory  = async (hospitalId: string) => {
     total: 4,
     // available: 63,
     unit: 'MT'
-    
-  }, {
-    resourceType: "Handwash",
-    total: 4,
-    available: 65
   }];
+}
+
+export interface HospitalAddress {
+  street: string;
+  city: string;
+  zip: number;
+  country: string;
+}
+
+export const fetchHospitalAddress = async (hospitalId: string) => {
```

Comment:
I'd confirm it with @kaashmonee and @devbhavikjain, but for simplicity, I'd expect the hospital address to be a part of the general Hospital resource, not something exposed with a separate endpoint.